install/kubernetes/helm/istio/charts/security/templates/clusterrole.yaml

```diff
@@ -15,7 +15,7 @@ rules:
   resources: ["secrets"]
   verbs: ["create", "get", "watch", "list", "update", "delete"]
 - apiGroups: [""]
-  resources: ["serviceaccounts", "services"]
+  resources: ["serviceaccounts", "services", "namespaces"]
```

Comment:
Mind putting some comments on the need of namespaces and service accounts? at least for serviceaccounts and services.

for `services` , @myidpt can you remind me why we require this privilege? I think even for mesh expansion, only pilot needs to read the annotation for secure naming, right?

Reply:
Citadel needs get and watch permissions for` serviceaccounts` so that it can be updated when a serviceaccount is created and then decide whether to create a corresponding secret. Needs permissions on `namespaces` because the namespace labels are what determine whether the secret should be created.

Reply:
Oh yeah, I understand, I'm saying should we put that as comment in this YAML file? Such that we can track the ACL matches the minimal requirements of the implementation as time pass by...

Reply:
Ah I see, that would definitely make sense to do.